File: 01-algaritmos-basicos/03-memoria_estrutura_repeticao/exercicios/em_C/fatorial.c
#include <stdio.h>

int main() {
	
	printf("Exercicio 2\n\n");
	
	printf("Desenvolva o codigo em C de um programa que recebe uma variavel inteira n e ");
	printf("imprime na tela o valor de n!\n\n");

	int n;
	int soma = 0;
	
	printf("Digite um numero: ");
	scanf("%d", &n);	
	soma = n;
	
	for(int i = n; i > 1; i--) {

		
		soma *= i;
		
	}

	printf("O fatorial de %d: %d", n, soma);
	
	return 0;
}

File: 01-algaritmos-basicos/03-memoria_estrutura_repeticao/exercicios/em_java/Fatorial.java
import java.util.Locale;
import java.util.Scanner;

public class Fatorial {

	public static void main(String[] args) {
	
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Exercicio 2 - Fatorial\n");
		
		int n;
		double soma;
		System.out.print("Digite um numero: ");
		n = sc.nextInt();
		soma = n;
		for( int i = n; i >= 1; i--) {
			
			soma *= i;
			
		}
		System.out.printf("O fatorial de %d e: %.2f", n, soma);
	}
	
}